presidio-image-redactor/presidio_image_redactor/image_pii_verify_engine.py

```diff
@@ -25,7 +27,11 @@ def __init__(self, image_analyzer_engine: Optional[ImageAnalyzerEngine] = None):
         self.image_analyzer_engine = image_analyzer_engine
```

Comment:
Can we use the super `__init__` instead?

Reply:
Yes, will remove in the next commit, verify that no issues arise, and then resolve this thread.

Reply:
@omri374 - Feedback addressed and all tests pass. Resolving this thread.